### PR TITLE
[BPD-3508] Add missing 'feeds.txt' file to gtfs merger

### PR DESCRIPTION
**Pull request by pbedn**

### CODE REVIEW

File: src/Command/Merge.php [hunk 1]
```diff
@@ -167,6 +171,12 @@ private function processGtfs($file, ProgressBar $progress)
         $zip = new ZipArchiveAdapter($file);
 
         foreach ($mergers as $merger => $subfile) {
+            if ($subfile === 'feed_info.txt') {
+                if ($this->feed_info_already_seen) {
+                    continue;
```
Comment (stekycz) [inline src/Command/Merge.php]: Why is this needed? `feed_info.txt` can be multiple times in GTFS?

File: tests/ConversionTest.php [hunk 2]
```diff
@@ -0,0 +1,47 @@
+<?php
+use PHPUnit\Framework\TestCase;
+
+
+class MergeTest extends TestCase
+{
+    public function testGtfsMerging()
+    {
+        $cache_dir = "tests/.cache";
```
Comment (stekycz) [inline tests/ConversionTest.php]: I would prefer to use `__DIR__ . '/tests/.cache'`

File: tests/ConversionTest.php [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+        if (!is_dir($cache_dir))
+            mkdir($cache_dir);
```
Comment (stekycz) [inline tests/ConversionTest.php]: Could this fail if the path would be a file? ;-)

File: tests/ConversionTest.php [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+
+        $input_a = "tests/assets/inputA.zip";
+        $input_b = "tests/assets/inputB.zip";
+        $test_output = "tests/.cache/output.zip";
```
Comment (stekycz) [inline tests/ConversionTest.php]: Use `__DIR__` pls

File: tests/ConversionTest.php [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+                    }
+                }
+            }
+        }
+        zip_close($zip);
+        unlink($test_output);
+        rmdir($cache_dir);
+
+        $this->assertEquals(9, $count);
+        $this->assertEquals(3, count(explode(PHP_EOL, $contents)));
```
Comment (stekycz) [inline tests/ConversionTest.php]: I am not sure if I understand well what this test does. Does it test that in the result after merging there is just a few of all mergeable items? I would prefer asserting the equality of an array for exact values.

File: src/Command/Merge.php (hunk 1 above)
Comment (pbedn) [inline src/Command/Merge.php]: Yes, there can be more rows inside feed_info, but because we do not use agency_name (connectio between feed_info.txt and agency.txt) all rows usually are the same (typical gtfs-merger usage)

File: tests/ConversionTest.php (hunk 2 above)
Comment (pbedn) [inline tests/ConversionTest.php]: If i change to that I get:
mkdir(): No such file or directory
What is wrong?
`        $cache_dir = __DIR__ . "/tests/.cache";
        if (!is_dir($cache_dir))
            mkdir($cache_dir);`

Comment (stekycz) [inline tests/ConversionTest.php]: I am not sure. Probably `__DIR__ . '/.cache'` could be enough since `__DIR__` means always the directory where the current file is.

File: tests/ConversionTest.php (hunk 3 above)
Comment (pbedn) [inline tests/ConversionTest.php]: yes, that would fail.. should I check for file first, then try to remove that and create dir instead?

File: tests/ConversionTest.php (hunk 5 above)
Comment (pbedn) [inline tests/ConversionTest.php]: firstly I wanted to assert that the number of all files is 9 (8 + feed_info)
secondly I want to check that there are only 3 rows inside feed_info: 
1. "title"
2. row with data
3. EOL

File: tests/ConversionTest.php (hunk 3 above)
Comment (stekycz) [inline tests/ConversionTest.php]: I think that better solution is to check if that exists. If so, then check if it is a file. If so then throw an exception and let the script die because the file could be important (do not know why right now but better to delete it manually instead of automatically delete something important). If it does not exists just create the directory. Maybe you could use second parameter for `mkdir` to create whole path if something is missing ;-)

File: tests/ConversionTest.php (hunk 2 above)
Comment (pbedn) [inline tests/ConversionTest.php]: my __ DIR __ is "/home/data/gtfs-merger/tests" that is why it fails
When I run tests from PhpStorm from that directory it is ok (without __ DIR __)
What is the best practice

File: tests/ConversionTest.php (hunk 5 above)
Comment (stekycz) [inline tests/ConversionTest.php]: Alright. So the second assert I would like to be `assertSame` for the same string as should be in the `feed_info.txt`. I think that it could get different content for different zip file order on the input.

File: tests/ConversionTest.php (hunk 2 above)
Comment (stekycz) [inline tests/ConversionTest.php]: The best practice is using `__DIR__` :-) As I mentioned above, the shorter variant must work.

**Comment by pbedn** [comment]
@stekycz All requested changes are corrected
